### DISCUSSION TITLE
[Frontend] SSR with Qwik on Cloudflare Pages

### DISCUSSION THREAD

**Discussion by guilhermegregio** [realworld-apps/realworld#985] [WIP Implementations]
Hello, we working on SSR with Qwik and Cloudflare pages.

Here is my repository: https://github.com/gang-of-front/realworld-qwik
Publish URL: https://gangoffront-realworld-qwik.pages.dev/

**Comment by geromegrignon** [comment]
Hi, thanks for using this project.
Please mind that as stated in our rules, we encourage collaboration if there is already a work in progress on a given stack as we don't accept similar implementations on CodebaseShow.
As @ranwahle is already actively working on an SSR Qwik implementation, I invite you to contact them to work together.
